ui/app/models/pki/issuer.js

```diff
@@ -1,6 +1,8 @@
-import Model, { attr } from '@ember-data/model';
-import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+import { attr } from '@ember-data/model';
 import { withModelValidations } from 'vault/decorators/model-validations';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
+import PkiCertificateBaseModel from './certificate/base';
```

Comment:
dust 🧹 I think it's nice to see the model import at the top of the file, but total nit